Subject: Add missing manpages
Author: Michael van der Kolff <mvanderkolff@gmail.com>
Forwarded: no

Just some help2man manpages.  Not worth bothering upstream with, since they're total crap.
--- /dev/null
+++ b/pbmtools/jbgtopbm85.1
@@ -0,0 +1,49 @@
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.40.4.
+.TH JBIGTOPBM "1" "April 2012" "JBIGtoPBM converter 2.0 (T.85 version) --" "User Commands"
+.SH NAME
+JBIGtoPBM \- manual page for JBIGtoPBM converter 2.0 (T.85 version) --
+.SH DESCRIPTION
+JBIGtoPBM converter 2.0 (T.85 version) \fB\-\-\fR
+reads a bi\-level image entity (BIE) as input file
+.PP
+usage: /usr/bin/jbgtopbm85 [<input\-file> | \-  [<output\-file>]]
+.PP
+options:
+.TP
+\fB\-x\fR number
+maximum number of pixels per line for which memory
+is allocated (default: 8192)
+.TP
+\fB\-y\fR number
+interrupt decoder after this number of lines
+.TP
+\fB\-B\fR number
+input buffer size
+.PP
+reads a bi\-level image entity (BIE) as input file
+.PP
+usage: /usr/bin/jbgtopbm85 [<input\-file> | \-  [<output\-file>]]
+.PP
+options:
+.TP
+\fB\-x\fR number
+maximum number of pixels per line for which memory
+is allocated (default: 8192)
+.TP
+\fB\-y\fR number
+interrupt decoder after this number of lines
+.TP
+\fB\-B\fR number
+input buffer size
+.SH "SEE ALSO"
+The full documentation for
+.B JBIGtoPBM
+is maintained as a Texinfo manual.  If the
+.B info
+and
+.B JBIGtoPBM
+programs are properly installed at your site, the command
+.IP
+.B info JBIGtoPBM
+.PP
+should give you access to the complete manual.
--- /dev/null
+++ b/pbmtools/pbmtojbg85.1
@@ -0,0 +1,65 @@
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.40.4.
+.TH PBMTOJBIG "1" "April 2012" "PBMtoJBIG converter 2.0 (T.85 version) --" "User Commands"
+.SH NAME
+PBMtoJBIG \- manual page for PBMtoJBIG converter 2.0 (T.85 version) --
+.SH DESCRIPTION
+PBMtoJBIG converter 2.0 (T.85 version) \fB\-\-\fR
+creates bi\-level image entity (BIE) as output file
+.PP
+usage: /usr/bin/pbmtojbg85 [<options>] [<input\-file> | \-  [<output\-file>]]
+.PP
+options:
+.TP
+\fB\-s\fR number
+height of a stripe
+.TP
+\fB\-m\fR number
+maximum adaptive template pixel horizontal offset (default 8)
+.TP
+\fB\-p\fR number
+options byte value: add TPBON=8, LRLTWO=64
+(default 8 = TPBON)
+.TP
+\fB\-C\fR string
+add the provided string as a comment marker segment
+.TP
+\fB\-Y\fR yi yr
+announce in header initially the larger image height yi
+and then announce after line yr has been encoded the real height
+using NEWLEN marker (for testing NEWLEN and VLENGTH=1 function)
+.PP
+creates bi\-level image entity (BIE) as output file
+.PP
+usage: /usr/bin/pbmtojbg85 [<options>] [<input\-file> | \-  [<output\-file>]]
+.PP
+options:
+.TP
+\fB\-s\fR number
+height of a stripe
+.TP
+\fB\-m\fR number
+maximum adaptive template pixel horizontal offset (default 8)
+.TP
+\fB\-p\fR number
+options byte value: add TPBON=8, LRLTWO=64
+(default 8 = TPBON)
+.TP
+\fB\-C\fR string
+add the provided string as a comment marker segment
+.TP
+\fB\-Y\fR yi yr
+announce in header initially the larger image height yi
+and then announce after line yr has been encoded the real height
+using NEWLEN marker (for testing NEWLEN and VLENGTH=1 function)
+.SH "SEE ALSO"
+The full documentation for
+.B PBMtoJBIG
+is maintained as a Texinfo manual.  If the
+.B info
+and
+.B PBMtoJBIG
+programs are properly installed at your site, the command
+.IP
+.B info PBMtoJBIG
+.PP
+should give you access to the complete manual.
